src/main/java/org/opentripplanner/framework/graphql/scalar/DateScalarFactory.java

```diff
@@ -19,10 +19,10 @@ public class DateScalarFactory {
 
   private DateScalarFactory() {}
 
-  public static GraphQLScalarType createDateScalar() {
+  public static GraphQLScalarType createDateScalar(String scalarName) {
     return GraphQLScalarType
       .newScalar()
-      .name("Date")
+      .name(scalarName)
       .description(DOCUMENTATION)
```

Comment:
Btw if I remember correctly, this overrides the documentation from the schema file.

Reply:
Fixed in https://github.com/opentripplanner/OpenTripPlanner/pull/5869/commits/9cfc1d7b8f2bd8f3a36504e70a02465705376cd3